.github/workflows/pipeline_run_from_notebook.yaml

```diff
@@ -69,7 +61,7 @@ jobs:
         kubectl apply -f tests/gh-actions/kf-objects/notebook.test.kubeflow-user-example.com.yaml
```

Comment:
@hansinikarunarathne after line 61, would be suitable to add the following command before the wait command:
```shell
kubectl describe Notebook -n kubeflow-user-example-com test
```

Reply:
You can also try this separately in your own PR, this way you can both experiment at the same time.

Reply:
Yes, I could make a PR push my change for the cronjob meanwhile.